.github/workflows/tests.yml

```diff
@@ -9,38 +9,26 @@ on:
 jobs:
   TenSEAL:
 
-    runs-on: ubuntu-latest
+    runs-on: ${{ matrix.os }}
     strategy:
       matrix:
         python-version: [3.6, 3.7, 3.8]
+        os: [macos-latest, ubuntu-latest]
 
     steps:
     - uses: actions/checkout@v2
     - name: Set up Python ${{ matrix.python-version }}
       uses: actions/setup-python@v1
       with:
         python-version: ${{ matrix.python-version }}
-    - name: Install dependencies
-      run: |
-        sudo apt update
-        sudo apt install cmake clang libprotobuf-dev protobuf-compiler
-        python -m pip install --upgrade pip
-        pip install -r requirements_dev.txt
-    - name: Lint with flake8
-      run: |
-        # stop the build if there are Python syntax errors or undefined names
-        flake8 --config=.flake8 --exclude=tenseal/sealapi,tests/sealapi .
-        black --check --exclude "sealapi" tenseal/ tests/
-        # exit-zero treats all errors as warnings.
-        flake8 . --count --exit-zero --max-complexity=10 --max-line-length=100 --statistics
-    - name: Lint with clang-format
-      run: |
-        # lint files (all .cpp and .h files) inplace
-        find ./tenseal/  \( -iname *.h -o -iname *.cpp \) -a -not -path "*sealapi*" | xargs clang-format -i -style='file'
-        # print changes
-        git diff
-        # already well formated if 'git diff' doesn't output
-        ! ( git diff |  grep -q ^ ) || return 1
+    - name: ubuntu-build
+      run: chmod +x .github/workflows/scripts/run_build_ubuntu.sh
```

Comment:
why chmod +x ? you can commit the flag

Reply:
without it's saying permission denied

Reply:
you can run chmod +x on your machine and commit the flag

Reply:
ok